corehq/apps/hqwebapp/session_details_endpoint/views.py

```diff
@@ -69,6 +72,14 @@ def post(self, request, *args, **kwargs):
             domains.update(EnterprisePermissions.get_domains(member_domain))
 
         enabled_toggles = toggles_enabled_for_user(user.username) | toggles_enabled_for_domain(domain)
+        enabled_previews = previews_enabled_for_domain(domain)
+        end_time = datetime.now()
+        metrics_histogram("commcare.session_details.processing_time",
+                      int((end_time - start_time).total_seconds() * 1000),
+                      bucket_tag='duration_bucket',
+                      buckets=(100, 250, 500, 1000, 2000, 3000, 4000),
```

Comment:
like the `limit_domains` comment - we get charged based on the number of unique tag combinations used, and datadog histograms actually turn buckets into tags, so using 7 divisions can actually be significant.  Is there a smaller number of splits that would still be useful?

Reply:
Ah, that's good to know. [fa1885c](https://github.com/dimagi/commcare-hq/pull/33955/commits/fa1885c1e5b3bd5ee052a27213b9e451700610a6)

Based on what i've seen on the past 3 days on staging, 3 buckets would be enough. 
0 - 250ms: fast
250ms - 1 sec: med slow
1sec - 3 sec: slow
![Screen Shot 2024-01-08 at 6 42 59 PM](https://github.com/dimagi/commcare-hq/assets/88759246/de65f720-0265-4e57-b2a1-4787c20ad3da)